google/cloud/tpu_v2alpha1/__init__.py

```diff
@@ -53,7 +53,6 @@
     OperationMetadata,
     QueuedResource,
     QueuedResourceState,
-    ResetQueuedResourceRequest,
```

Comment:
This is a breaking change however it part of an alpha API and was only recently added in version [1.10.0](https://github.com/googleapis/python-tpu/pull/225)